Mathlib/CategoryTheory/Limits/HasLimits.lean

```diff
@@ -312,22 +312,24 @@ theorem limit.lift_extend {F : J ⥤ C} [HasLimit F] (c : Cone F) {X : C} (f : X
     limit.lift F (c.extend f) = f ≫ limit.lift F c := by aesop_cat
 #align category_theory.limits.limit.lift_extend CategoryTheory.Limits.limit.lift_extend
 
+/-- Transport a `LimitCone` along an equivalence. -/
+def LimitCone.transport {F G : J ⥤ C} (α : F ≅ G) (c : LimitCone F) : LimitCone G where
+  cone := (Cones.postcompose α.hom).obj c.cone
+  isLimit :=
+    { lift := fun s => c.isLimit.lift ((Cones.postcompose α.inv).obj s)
+      fac := fun s j => by
+        rw [Cones.postcompose_obj_π, NatTrans.comp_app, ← Category.assoc, c.isLimit.fac]
+        simp
+      uniq := fun s m w => by
+        apply c.isLimit.hom_ext; intro j
+        rw [c.isLimit.fac, Cones.postcompose_obj_π, NatTrans.comp_app, ← NatIso.app_inv,
+          Iso.eq_comp_inv]
+        simpa using w j }
```

Comment:
We already have this definition:
```suggestion
  isLimit := (IsLimit.postcomposeHomEquiv _ _).symm c.isLimit
```
So that it would be better to refactor the proof of `hasLimitOfIso` below using this.
(Also, I would think that most of us would agree we do not want to develop `LimitCone` API, but rather "stating" that certain cones are limit.)

Reply:
Nice catch, done